### PR TITLE
ONI changed the way they use animations in MODs

### PR DESCRIPTION
**Pull request by phreak2599**
I am unable to find out how they changed the mod animations, so I changed the code to use the standard liquid and gas animations.

I have not finished the research yet on my current play through, so this is untested.

### CODE REVIEW

**Comment by phreak2599** [comment]
> 
> 
> Do you think you can upload a compiled mod either via a branch or on steam. I don't know how to compile your source code into a dll that the game would read.

Sure I can. Please see https://github.com/phreak2599/ONI-Mods/releases/tag/0.1a

**Comment by Ashotep** [comment]
Thank you.

File: src/CrystalBiome/CrystalBiome.csproj [hunk 1]
```diff
@@ -34,13 +34,13 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="0Harmony">
-      <HintPath>..\lib\0Harmony.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Steam\SteamApps\common\OxygenNotIncluded\OxygenNotIncluded_Data\Managed\0Harmony.dll</HintPath>
```
Comment (TZer0) [inline src/CrystalBiome/CrystalBiome.csproj]: All of this will break it for the plugin dev.

Comment (phreak2599) [reply src/CrystalBiome/CrystalBiome.csproj]: Yep, my mistake. Thanks for pointing that out. I am quite new to github, and a bit of a noob with git in general.

Comment (TZer0) [reply src/CrystalBiome/CrystalBiome.csproj]: No worries. :)

**Comment by HyperWilson** [comment]
Hey, i don't really know where to post this, took me a longer than it should have to find this thread, but i encountered an error an hour ago and thought i'd post it here, judging by the report... it's something about the info screen???

NullReferenceException: Object reference not set to an instance of an object
  at TeleStorage.TeleStoragePatches+SimpleInfoScreen_RefreshStorage_Patch.Postfix (SimpleInfoScreen __instance, UnityEngine.GameObject ___storagePanel, UnityEngine.GameObject ___selectedTarget, System.Collections.Generic.Dictionary`2[System.String,UnityEngine.GameObject]& ___storageLabels) [0x00001] in <2f3156c3da7e4e5cbd409ef290d3b495>:0 
  at (wrapper dynamic-method) SimpleInfoScreen.RefreshStorage_Patch1(object)
  at SimpleInfoScreen.Refresh (System.Boolean force) [0x000a3] in <1451e1c3b7a64dd1bc2ed2e3a3ee7859>:0 
  at SimpleInfoScreen.OnRefreshData (System.Object obj) [0x00000] in <1451e1c3b7a64dd1bc2ed2e3a3ee7859>:0 
  at SimpleInfoScreen+<>c.<.cctor>b__60_0 (SimpleInfoScreen component, System.Object data) [0x00000] in <1451e1c3b7a64dd1bc2ed2e3a3ee7859>:0 
  at EventSystem+IntraObjectHandler`1[ComponentType].Trigger (UnityEngine.GameObject gameObject, System.Object eventData) [0x0001e] in <5e8d06b8fe0d451c94539ae3b47c5a2e>:0 
  at EventSystem.Trigger (UnityEngine.GameObject go, System.Int32 hash, System.Object data) [0x0004f] in <5e8d06b8fe0d451c94539ae3b47c5a2e>:0 
  at KMonoBehaviour.Trigger (System.Int32 hash, System.Object data) [0x00026] in <5e8d06b8fe0d451c94539ae3b47c5a2e>:0

**Comment by HyperWilson** [comment]
Okay, so i just got the exact same crash, my game froze for a split second and i clicked on some chlorine and immediately got the error, i'm not entirely sure if this is true, but i think the mod is trying to refresh the info screen on the (chlorine, for this instance) but that particular info screen never held that information, thus resulting in the crash.

**Comment by HyperWilson** [comment]
I also forgot to mention, this is an error for the telestorage mod, my bad.

**Pull request merged**